tc/sgx/trusted_worker_manager/enclave/work_order_processor.h

```diff
@@ -31,6 +32,7 @@ namespace tcf {
             ByteArray CreateErrorResponse(int err_code, const char* err_message);
             ByteArray Process(EnclaveData* enclaveData, std::string json_str);
             std::string ext_work_order_data;
+            EnclaveType enclave_type = SINGLETON_ENCLAVE;
```

Comment:
Can we initialize enclave_type  in constructor or as constructor initializer list